examples/http_client.py

```diff
@@ -16,6 +16,7 @@
 from neubot_runtime.http_client import HttpClient
 from neubot_runtime.http_message import HttpMessage
 from neubot_runtime.poller import Poller
+from neubot_runtime import compat23
```

Comment:
Why do you import `compat23` here? It seems you are not going to use it afterwards.